index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>Document</title>
+</head>
+<body>
+    <div class="main-nav">
+        <nav>
+            <ul class="nav">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">about</a></li>
+
+            </ul>
+        </nav>
+    </div>
+
+    <div class="main-body">
+
+        <div class="main-title">
+            <h1 id="grading-element-five">
+                <span style="color:cornflowerblue">D</span>
+                <span style="color:red ">O</span>
+                <span style="color:yellow ">O</span>
+                <span style="color: cornflowerblue">G</span>
+                <span style="color:green ">L</span>
+                <span style="color: red">E</span>
+            </h1>
+        </div>
+
+
+
+        <form  action="http://localhost:3000/form" name="grading-element-four" method="POST">
+
+            <input class="bar-form" name="testOne"  value="OnValue" type="text" required >
+
+            <input class="inputHide" value="value" name="testTwo"  type="text" required>
+
+            <input class="inputHide" value="value" name="TestThree" type="text" required>
+            <input type="submit"/>
+         </form>
+
+        <button onclick="Onvisible()">Luck</button>
+
+        <br>
+
+        <div id="grading-element-three-consequence" hidden >
```

Comment:
stays `hidden` even after i click `#grading-element-three`

Reply:
I fixed the error